### DISCUSSION TITLE
Can't pick more than one subtitle from source anymore

### DISCUSSION THREAD

**Discussion by KaoruGreen** [RandomEngy/VidCoder#1143] [Q&A]
I'm not sure whether this happened (or changed) with a recent update, but for some reason I'm not able to pick more than one subtitle to encode from the source file anymore. This used to be possible until recently, but now Vidcoder outright refuses to do this anymore, and I can't figure out why this is. I searched through the settings many many many times whether there's a setting that I accidentally botched up or if there's a box that I need to (un)tick, but I continue to be at a loss.

Does anyone else have an idea or even a solution to this problem? What am I missing?

**Comment by RandomEngy** [comment]
Some container and subtitle combinations must be burned in; they don't support soft subtitles. The one that usually comes up with PGS subtitles with an MP4 container output.
